devel/boost/Portfile

```diff
@@ -255,6 +256,15 @@ http://www.boost.org/doc/libs/${distver}/libs/regex/doc/html/boost_regex/ref/sub
     }
 }
 
+variant numpy description {Enable Numpy support} {
```

Comment:
This should be a subport, not a variant ... or at least that's my interpretation of our discussion on the main ticket ... yes? Moving to being a subport changes most of this commit.

Reply:
I wasn't sure if that's the way you want to go. My initial thought was to go with the variant, since that avoids all the recompilation, but if you like the subport approach better I can rewrite the PR to do that. Do you want a separate PR or should I just push another commit to this one?

Reply:
Generally it's better to use the same PR if it's about the same issue. Two PRs make sense for different issues or in cases when for example two non-committers cannot push to the same branch (a non-committer would want to provide a better solution than the one provided in an existing PR; or if there would be no consensus about which of two proposals is better).